checks.d/win32_event_log.py

```diff
@@ -155,16 +178,43 @@ def _msg_title(self):
 
     @property
     def _msg_text(self):
-        msg_text = ""
-        if 'Message' in self.event:
-            msg_text = "{message}\n".format(message=self.event['Message'])
-        elif 'InsertionStrings' in self.event:
-            msg_text = "\n".join([i_str for i_str in self.event['InsertionStrings']
-                                  if i_str.strip()])
-
-        if self.notify_list:
-            msg_text += "\n{notify_list}".format(
-                notify_list=' '.join([" @" + n for n in self.notify_list]))
+        """
+        Generate the event's body to send to Datadog.
+
+        Consider `event_format` parameter:
+        * Only use the specified list of event properties.
+        * If unspecified, default to the EventLog's `Message` or `InsertionStrings`.
+        """
+        msg_text = u""
+
+        if self._format:
+            msg_text_fields = ["%%%\n```"]
+
+            for event_property in self._format:
+                property_value = self.event.get(event_property)
+                if property_value is None:
+                    self.log.warning(u"Unrecognized `%s` event property.", event_property)
+                    continue
+                msg_text_fields.append(
+                    u"{property_name}: {property_value}".format(
+                        property_name=event_property, property_value=property_value
+                    )
+                )
+
+            msg_text_fields.append("```\n%%%")
+
+            msg_text = "\n".join(msg_text_fields)
+        else:
+            # Override when verbosity
+            if 'Message' in self.event:
+                msg_text = "{message}\n".format(message=self.event['Message'])
+            elif 'InsertionStrings' in self.event:
+                msg_text = "\n".join([i_str for i_str in self.event['InsertionStrings']
+                                      if i_str.strip()])
+
+            if self.notify_list:
+                msg_text += "\n{notify_list}".format(
+                    notify_list=' '.join([" @" + n for n in self.notify_list]))
```

Comment:
The `notify_list` should probably be added to the message body even when a specific `_format` is specified, right? (even though that option is not documented and should probably be deprecated)